### DISCUSSION TITLE
About samplers

### DISCUSSION THREAD

**Discussion by tetsuoo-online** [carson-katri/dream-textures#102] [Q&A]
Hi, I'm posting here because I'm not sure it's an issue, could be the user :)
I just noticed the samplers so I was playing around a bit, then... I decided to replicate the same generated image in Colab [https://colab.research.google.com/drive/1jUwJ0owjigpG-9m6AI_wEStwimisUE17](url)
Using the same model, same seed, steps etc I finally got about 99% the same result :
- in Dream-Textures by using DDIM "sampler"
-  in Colab by using K-LMS or PNDM "scheduler"

Now assuming samplers and schedulers are the same thing (right ?), I'm wondering which one is wrong ? x)
If I use DDIM in Colab I will get a completely different result

**Comment by carson-katri** [comment]
This is probably because of changes to the model in InvokeAI, the fork of stable diffusion this addon uses as a submodule. So it isn’t 100% compatible with other forks.

FYI, all samplers should eventually resolve to the same image given enough steps.
